templates/RangeFacet.mustache

```diff
@@ -0,0 +1,6 @@
+<div>
+	{{msg-min}}: <input name="min" type="text" value="{{min}}" />
+</div>
+<div>
+	{{msg-max}}: <input name="max" type="text" value="{{max}}" />
```

Comment:
Ah, so these min and max are `currentMin` and `currentMax`. That's not obvious in the caller code, where I was thinking they represent `rangeMin` and `rangeMax`. I'm happy these are the current values, since it's not clear we will need the boundaries of the range.

Reply:
Boundaries might be a UX requirement if we do the slider, otherwise there's no limit to where you slide. But otherwise as far as the query goes, there is no technical requirement for boundaries. If `rangeMax` was calculated as 1000 by ElasticSearch (i.e. the matching items only go up to 1000) and you enter 2000, then there should be no additional matching items anyway.